plugins/cybereason/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: cybereason
 title: Cybereason
 description: The Cybereason platform provides military-grade cyber security with real-time awareness and detection. Respond to threats and remediate in seconds using the Cybereason plugin
-version: 2.1.0
+version: 2.2.0
 supported_versions: ["All"]
```

Comment:
Is there a version of Cyberreason that this was tested on that can be added here as the supported version ?